src/tasks/components/TaskForm.scss

```diff
@@ -32,3 +32,11 @@
   flex: 1 0 60px;
   padding-left: 11px;
 }
+
+.task-runs-row-btn {
+  margin-right: 10px !important;
```

Comment:
We should avoid using `!important` at all costs since it tends to make things confusing to debug later on. Also can this be a clockface variable for consistency?